src/hotspot/share/oops/cpCache.cpp

```diff
@@ -690,7 +687,7 @@ void ConstantPoolCacheEntry::print(outputStream* st, int index, const ConstantPo
                flag_state(), is_final(), is_volatile(), field_index());
     st->print_cr(" - tos: %s\n - final: %d\n - volatile: %d\n - field index: %04x",
           type2name(as_BasicType(flag_state())), is_final(), is_volatile(), field_index());
-    //st->print_cr(" - is resolved: %s", pool->tag_at(constant_pool_index()->is_unresolved_klass()) ? "false" : "true");
+    st->print_cr(" - is resolved: %s", pool->tag_at(constant_pool_index()).is_unresolved_klass() ? "false" : "true");
```

Comment:
This is confusing and doesn't really help.  "is_resolved" means that the bytecode_1 or bytecode_2 is filled in in the indices field.  The constant pool index here is a JVM_CONSTANT_Fieldref which points to JVM_CONSTANT_NameAndType and JVM_CONSTANT_{UnresolvedClass,Class,UnresolvedClassInError}.  I think this would always print false. I would just leave this line out.